src/lab/Graphs/LineAreaGraph/styles.ts

```diff
@@ -68,6 +68,11 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginTop: "1rem",
     padding: "1rem",
     backgroundColor: `${theme.palette.cards.background} !important`,
+    "-o-transition": "all 0.3s ease-out",
+    "-ms-transition": "all 0.3s ease-out",
```

Comment:
in my knowledge mui adds the vendor prefixes.
You can check once...and if that's the case only keep the single transition property

Reply:
This tooltip is a visx component

Reply:
The styling is still using Mui right...so I think you can keep only the main transition property

Reply:
Removed transiton. Tooltip code modified. PTAL